src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -2,11 +2,15 @@
 
 {% load staticfiles %}
 
+{% block metatitle %}
+    <title>Model My Watershed Micro Site Storm Model</title>
```

Comment:
Updated to have ':' as separator. Sorry, should've done that in the first place.